message/router/middleware/retry.go

```diff
@@ -10,6 +10,8 @@ import (
 
 type OnRetryHook func(retryNum int, delay time.Duration)
 
+// Retry provides a middleware that retries the handler if errors are returned.
+// The retry behaviour is configurable, with exponential backoff and maximum elapsed time.
 type Retry struct {
 	MaxRetries int
```

Comment:
it would be nice to have docs also for params

Reply:
you do that, you wrote the middleware xD
but more seriously, I'll try to come up with something

Reply:
I've updated the retry middleware to use cenkalti/backoff
4538ec1ea4a94403d70a976f5ab9b828091840f7